imageio_ffmpeg/_io.py

```diff
@@ -99,6 +99,9 @@ def read_frames(path, pix_fmt="rgb24", bpp=3, input_params=None, output_params=N
             This depends on the given pix_fmt. Default is 3 (RGB).
         input_params (list): Additional ffmpeg input command line parameters.
         output_params (list): Additional ffmpeg output command line parameters.
+        bits_per_pixel (int): The number of bits per pixel in the output frames.
+            This depends on the given pix_fmt. Yuv format can have 12 bits per pixel,
+            so bpp attribute useless
```

Comment:
This docstring looks unfinished?

It also looks a bit confusing to have two similar params, so we need to explain things a bit better. Could you move the bpp docstring to the end, add a note that `bits_per_pixel` is preferred, and that `bpp` is there for backwards compatibility?

Reply:
@almarklein I've updated docstring.